src/emails/emailAdminNewDeal.tsx

```diff
@@ -0,0 +1,85 @@
+import * as React from 'react'
+import {
+  Body,
+  Container,
+  Head,
+  Heading,
+  Html,
+  Img,
+  Preview,
+  Section,
+  Text,
+  Tailwind,
+  Link,
+} from '@react-email/components'
+
+interface EmailTemplateProps {
+  dealId: string
+  appDomain: string
+}
+
+export const emailAdminNewDeal = (props: EmailTemplateProps) => (
+  <Tailwind>
+    <Html lang="en">
+      <Head />
+      <Preview>New Deal Submitted</Preview>
+      <Body className="mx-auto bg-white p-4">
+        <Container>
+          <Section>
+            <Img
+              src={`${props.appDomain}/logo-teal.png`}
+              width="60"
+              alt="Deals for Devs"
+            />
+          </Section>
+          <Heading style={h1}>Admin - Review new deal</Heading>
+          <Section className="py-6">
+            <Link
+              href={`${props.appDomain}admin/dashboard/deals${props.dealId}`}
```

Comment:
Link is malformed. It should be `{`${props.appDomain}/dashboard/deals/${props.dealId}`}`